index.html

```diff
@@ -18,7 +204,12 @@
 
 
 <!-- JavaScript Files-->
-<script type='text/javascript' src='dist/app.js'></script>
 
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```

Comment:
We'll want to change the CDN reference to our local bootstrap module at `lib/node_modules/bootstrap/dist/js/bootstrap.min.js`